RallyHereIntegration/Source/RallyHereIntegration/Private/RH_FriendSubsystem.cpp

```diff
@@ -1712,7 +1765,7 @@ void URH_FriendSubsystem::StartBlockedRefreshTimer()
 
 	if (BlockedListPoller.IsValid())
 	{
-		BlockedListPoller->StartPoll(FRH_PollFunc::CreateUObject(this, &URH_FriendSubsystem::PollBlockedPlayers), PollTimerName);
+		BlockedListPoller->StartPoll(FRH_PollFunc::CreateUObject(this, &URH_FriendSubsystem::PollBlockedPlayers), PollTimerName, true);
```

Comment:
This is to make the presence system push presence data immediately upon user change, rather than waiting a delay cycle.